src/pages/signin/SignInPageLayout/index.js

```diff
@@ -35,34 +36,48 @@ const SignInPageLayout = (props) => {
         </SignInPageContent>
     );
 
+    const hasRedirect = !_.isEmpty(backgroundStyle.redirectUri);
+
+    const graphicLayout = (
+        <Pressable
+            style={[
+                styles.flex1,
+                StyleUtils.getBackgroundColorStyle(backgroundStyle.backgroundColor),
+            ]}
+            onPress={() => {
+                Link.openExternalLink(backgroundStyle.redirectUri);
+            }}
+            disabled={!hasRedirect}
+        >
+            <SVGImage
+                width="100%"
+                height="100%"
+                src={backgroundStyle.backgroundImageUri}
+                resizeMode={props.isMediumScreenWidth ? 'contain' : 'cover'}
+            />
+        </Pressable>
+    );
+
     if (props.isSmallScreenWidth) {
         return content;
     }
 
-    const hasRedirect = !_.isEmpty(backgroundStyle.redirectUri);
+    if (props.isMediumScreenWidth && props.windowHeight >= variables.minHeigthToShowGraphics) {
+        return (
```

Comment:
It's a typo here, that causing the issue. It should have been`minHeightToShowGraphics`

@aneequeahmad @Julesssss

Reply:
Easy to miss, I do this mistake all the time 😬 
Thanks for catching!

Reply:
@Santhosh-Sellavel should i raise a quick PR for this fix ?

Reply:
Ah I didn't see your comment @aneequeahmad. I'm on it already 👍

Reply:
@Julesssss Can't thank you enough 🙇

Reply:
Resolving, [raised PR](https://github.com/Expensify/App/pull/9129)